homeassistant/components/gtfs/sensor.py

```diff
@@ -488,26 +485,26 @@ def update(self) -> None:
             else:
                 trip_id = self._departure['trip_id']
                 if not self._trip or self._trip.trip_id != trip_id:
-                    _LOGGER.info("Fetching trip details for %s", trip_id)
+                    _LOGGER.debug("Fetching trip details for %s", trip_id)
                     self._trip = self._pygtfs.trips_by_id(trip_id)[0]
 
                 route_id = self._departure['route_id']
                 if not self._route or self._route.route_id != route_id:
-                    _LOGGER.info("Fetching route details for %s", route_id)
+                    _LOGGER.debug("Fetching route details for %s", route_id)
                     self._route = self._pygtfs.routes_by_id(route_id)[0]
 
             # Fetch agency details exactly once
             if self._agency is None and self._route:
                 try:
-                    _LOGGER.info("Fetching agency details for %s",
-                                 self._route.agency_id)
+                    _LOGGER.debug("Fetching agency details for %s",
```

Comment:
We should move this up out of the try... except.

Reply:
Good point, fixed!